src/types.ts

```diff
@@ -242,17 +243,15 @@ export interface ITransaction {
 	id: string;
 	token: string;
 	amount_usd: number;
-	amount_token: number | number[];
+	amount_token: string;
```

Comment:
please confirm this change

Reply:
Yes, this is handled everywhere, there wont be any problem for older txns too